### DISCUSSION TITLE
Cost setting for filament

### DISCUSSION THREAD

**Discussion by ro4short** [OrcaSlicer/OrcaSlicer#422] [General]
Where can I adjust the cost of filament?  I checked under Prefences and didn't see anything.  Also, didn't see anything when I first installed it.  TIA

**Comment by palumbus81** [comment]
When you open the settings of the filament, you can then set the "Price" as "money/KG".
![image](https://user-images.githubusercontent.com/123122038/222960152-8fbcd840-d417-451b-9f35-8042325bbaa2.png)

**Reply by ro4short** [reply]
Oh geez okay I found it. Thank you.  I kept clicking on the gear and didn't see the edit button to the right 
Thank you again

**Comment by 3Dlonra** [comment]
Thanks, I was looking for how to enter the cost of the material, with respect to the energy cost?
